### PR TITLE
have the set-up page for admin and the userManagement portion for admins

### PR DESCRIPTION
**Pull request by curiousYi**

### CODE REVIEW

File: server/app/routes/users/index.js [hunk 1]
```diff
@@ -19,12 +19,31 @@ router.get('/me/cart', function(req, res, next) {
...
         .then(user => {
             if (!user) res.sendStatus(404);
             else res.send(user.sanitize());
         })
         .catch(err => next(err));
 });
 
+//router for admin to search for Users to set them as admin
+
+router.get('/findUsers', function(req, res, next){
```
Comment (nrod80) [inline server/app/routes/users/index.js:32]: maybe make this: router.get('/', function(....?

File: browser/js/admins-only/admins-only.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    $scope.resetPassword = function(user) {
+
+        console.log('you hit resetPassword');
+        console.log(user)
+    }
+
+    $scope.deleteUser = function(user) {
+
+        console.log('you hit DeleteUser');
+        console.log(user)
```
Comment (nrod80) [inline browser/js/admins-only/admins-only.js:55]: lots of console.logs here

File: browser/js/admins-only/admins-factory.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+        return $http.get('/api/users/findUsers')
+                .then(function(arrayOfUsers){
+                    console.log('got something from adminsFactory findUsers')
+                    if(arrayOfUsers.length){
+                        return false
+                    }
+                    console.log('successfully hit adminsFactory find Users')
+                    console.log(arrayOfUsers);
+                    return arrayOfUsers.data;
```
Comment (nrod80) [inline browser/js/admins-only/admins-factory.js:15]: console logs

File: server/app/routes/users/index.js [hunk 4]
```diff
@@ -19,12 +19,31 @@ router.get('/me/cart', function(req, res, next) {
...
+
+    if (!req.user || !req.user.isAdmin){
+        return res.sendStatus(401);
+    }
+
+    console.log('you hit the req /api/findUsers' , req.data)
```
Comment (nrod80) [inline server/app/routes/users/index.js:38]: console.log

**Review by nrod80 — CHANGES_REQUESTED** [review]
Remove the extraneous console.logs and make the routes/users more RESTful

**Review by clocasto — APPROVED** [review]

**Review by nrod80 — APPROVED** [review]

**Review by curiousYi — COMMENTED** [review]
These changes look good, I will keep in mind the suggestion to change it to '/' but I think it's ok for now.

**Pull request merged**